testing/testcafe/tests/navigation/toolbar/multiline.ts

```diff
@@ -55,51 +49,48 @@ const themes = ['generic.light', 'generic.light.compact', 'material.blue.light',
       multiline: true,
       items: toolbarItems,
       rtlEnabled,
-    });
-  }).after(async (t) => {
-    await restoreBrowserSize(t);
+    }, false, '#toolbar');
```

Comment:
Let's move last 2 parameters to collecting object parameter. Or maybe it's better to exchange its positions and move to object parameter only animation disable (if we pass selector to element almost always)?

Reply:
createWidget function uses widely throught our tests. It will take a lot of time change the established way to pass a parameters to it. I am plan to try switch off animation for all testcafe tests in future merges.